src/database/models/userModel.ts

```diff
@@ -0,0 +1,23 @@
+// Import validation decorators from the class-validator library
+import { IsEmail, IsString, MinLength, MaxLength } from 'class-validator';
+
+// Define the user class that will have valid`ation rules for its property
+export class User {
+  // Apply the IsEmail decorator to validate the email property
+  // {} is an empty options object, followed by a custom error message if validation fails  
+  @IsEmail({}, { message: 'Email must be valid'})
```

Comment:
Don't include validation into models we will use a package called Joi validation to do soo later.